### DISCUSSION TITLE
How to set a shortcut for expanding and collapsing sidebar

### DISCUSSION THREAD

**Discussion by sourovch** [zen-browser/desktop#562] [Q&A]
hi I used firefox with an extension for that enables sidebar. I used to collapse and expend my sidebar using f1 key so it has became a habit. recently I moved to zen I couldn't find any options to change expend/collapse shortcut option. can any one let me know is there a way to do it?

**Comment by fviz** [comment]
Do you mean this? It's the very last option, was set by default to `Alt + P` on Windows.

![image](https://github.com/user-attachments/assets/6cbe89d9-4893-4152-8e64-fe3f031002c3)
